src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -226,4 +233,13 @@ th {
 	font-size: small;
 	margin-top: 16px;
 }
+.color-dot {
+	display: inline-block;
+	vertical-align: middle;
+	width: 14px;
+	height: 14px;
+	margin: 0 14px;
```

Comment:
Tiny detail, would look better with only `margin-right: 14px;` so the color dots are aligned with the text above.

Otherwise super nice! :)

Reply:
Thanks for the feedback. I removed the left margin and updated the screenshot.